CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 ## [2.1.0] - 2022-08-17
-### Added
+###
```

Comment:
```suggestion
### Added
```

Reply:
It's not. First time editing a chagelog and didn't notice that the header had also a title, sorry.